src/pages/Environments.js

```diff
@@ -539,21 +539,21 @@ export const Environments = ({ nav = undefined }) => {
     }, [makeMenuIcon('trash'), 'Delete'])
   }
 
-  const renderPauseButton = (computeType, compute) => {
+  const renderPauseButton = (computeType, compute, isGcp) => {
     const { status } = compute
 
     const shouldShowPauseButton =
-      Utils.cond(
-        [isApp(compute) && !_.find(tool => tool.appType && tool.appType === compute.appType)(appTools)?.isPauseUnsupported, () => true],
-        [isPauseSupported(getToolLabelFromRuntime(compute)) && currentUser === getCreatorForRuntime(compute), () => true],
-        () => false)
+    Utils.cond(
+      [isApp(compute) && !_.find(tool => tool.appType && tool.appType === compute.appType)(appTools)?.isPauseUnsupported, () => true],
+      [isPauseSupported(getToolLabelFromRuntime(compute)) && currentUser === getCreatorForRuntime(compute), () => true],
+      () => false)
 
-    return shouldShowPauseButton && h(Link, {
+    return shouldShowPauseButton && h(Link, { //TODO: IA-3993 enable pausing
       style: { marginRight: '1rem' },
-      disabled: !isComputePausable(computeType, compute),
-      tooltip: isComputePausable(computeType, compute) ?
+      disabled: !isGcp || !isComputePausable(computeType, compute),
+      tooltip: !isGcp ? 'Feature coming soon' : (isComputePausable(computeType, compute) ?
```

Comment:
1. Can we flatten all this conditionality into a single `Utils.cond`? If it's Azure, show NYI; if it's pausable, show 'Pause'; if it's not, show 'Cannot pause'.
2. Nit: Rather than not-GCE, can we be looking at is-Azure for the feature flag? For clarity.